src/pages/landing/ebooks/components/_introduction.js

```diff
@@ -109,13 +96,25 @@ const Introduction = ({ introImage, imageWidth, introPara, subPara, introList })
                 />
                 <MediaBody>
                     <Header as="h3" className="mt-0 intro-text">
-                        Introduction
+                        {localize('Introduction')}
                     </Header>
-                    <Text className="intro-desc">{introPara}</Text>
+                    <Text className="intro-desc" mb="20px">
+                        {introPara}
+                    </Text>
                     {subPara && <Text className="intro-desc">{subPara}</Text>}
                     {introList && (
                         <>
-                            <Header as="h5">In this book you’ll learn:</Header>
+                            <Header
+                                as="h5"
+                                type="sub-section-title"
+                                weight="bold"
+                                lh={1.5}
+                                color="#333333"
```

Comment:
Can we use the common variable for this? 🙏

Reply:
Okay done!